static/js/settings_org.js

```diff
@@ -846,10 +871,15 @@ export function register_save_discard_widget_handlers(container, patch_url) {
 
         for (let input_elem of properties_elements) {
             input_elem = $(input_elem);
-            if (check_property_changed(input_elem)) {
+            if (check_property_changed(input_elem, for_realm_default_settings)) {
                 const input_value = get_input_element_value(input_elem);
                 if (input_value !== undefined) {
-                    const property_name = input_elem.attr("id").replace("id_realm_", "");
+                    let property_name;
+                    if (for_realm_default_settings) {
+                        property_name = input_elem.attr("name");
```

Comment:
Same reason as for the `extract_property_name` function.

Reply:
And this can have perhaps a brief comment referencing the other comment, or duplicating it.